### PR TITLE
chore: add some tests

### PR DESCRIPTION
**Pull request by ofrobots**
Ref: https://github.com/ofrobots/require-so-slow/issues/8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,11 +14,14 @@
...
     "@types/npm-package-arg": "^5.1.0",
+    "@types/tape": "^4.2.32",
     "gts": "^0.5.4",
+    "source-map-support": "^0.5.4",
+    "tape": "^4.9.0",
     "typescript": "~2.6.1"
   },
   "scripts": {
-    "test": "node -e 0",
+    "test": "npm run compile && tape -r source-map-support/register build/test/test*.js",
```
Comment (JustinBeckwith) [inline package.json]: can we rely on `pretest` to do the compilation for us?

File: test/fixtures/modC.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import * as A from './modA';
+import * as B from './modB';
+
+export const name = 'modC';
```
Comment (JustinBeckwith) [inline test/fixtures/modC.ts]: newline!

File: test/test-shim.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+test('modules already in cached do not show up in trace', t => {
+  require('./fixtures/modC');
+  const events = perfTrace.getAndClearEvents();
+  t.equal(events.length, 1);
+  t.end();
+});
+
+test.skip('record a trace for a module that throws', t => {});
+
+test.skip('record a trace for a non-existent module', t => {});
```
Comment (JustinBeckwith) [inline test/test-shim.ts]: newline

**Review by JustinBeckwith — APPROVED** [review]

File: test/test-shim.ts (hunk 3 above)
Comment (ofrobots) [reply test/test-shim.ts]: Done.

File: test/fixtures/modC.ts (hunk 2 above)
Comment (ofrobots) [reply test/fixtures/modC.ts]: Done.

File: package.json (hunk 1 above)
Comment (ofrobots) [reply package.json]: Fixed.

**Pull request merged**